src/engine/source/cmds/include/cmds/graph.hpp

```diff
@@ -21,7 +21,7 @@ struct Options
 {
```

Comment:
Documentation should also be updated to 'policy'.

Reply:
done